migration/migrator/migrations/course/20210113174909_registration_subsection.py

```diff
@@ -0,0 +1,7 @@
+
+def up(config, database, semester, course):
+    database.execute('ALTER TABLE users ADD COLUMN registration_subsection character varying(255)')
+
+
+def down(config, database, semester, course):
+    database.execute('ALTER TABLE users DROP IF EXISTS registration_subsection')
```

Comment:
I don't think it should be necessary to delete this column in the down migration (we don't want to lose data if we temporarily revert this PR).